modules/git/blame.go

```diff
@@ -87,10 +87,11 @@ func (r *BlameReader) Close() error {
 
 // CreateBlameReader creates reader for given repository, commit and file
 func CreateBlameReader(repoPath, commitID, file string) (*BlameReader, error) {
-	_, err := OpenRepository(repoPath)
+	gitRepo, err := OpenRepository(repoPath)
```

Comment:
Just check if the repoPath exist?

Reply:
Well I suspect OpenRepository does a bit more than just check if the directory exists - however I agree this is silly.

I will have a look.

Reply:
TBH blame needs a total refactor. We never actually run this code not on a repository so this is somewhat redundant. Rather than refactor that here I think just ensuring that this is closed should be sufficient.